src/pages/Product.js

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import Sidebar from "../components/Sidebar"
```

Comment:
Suggestion :  make a index file for components too and import all components from there